src/ui/pages/ManageWallets/WalletGroup/WalletGroup.tsx

```diff
@@ -256,8 +250,16 @@ export function WalletGroup() {
               items={[
                 {
                   key: 1,
-                  to: `/backup-wallet?groupId=${walletGroup.id}&backupKind=verify`,
-                  onClick: needsBackup(walletGroup) ? openInTabView : undefined,
+                  to: isMnemonicGroup
+                    ? `/backup?groupId=${walletGroup.id}`
+                    : undefined,
+                  onClick: isMnemonicGroup
+                    ? (event) =>
+                        openHref(event, {
+                          windowType: 'tab',
+                          windowLayout: 'page',
+                        })
+                    : undefined,
```

Comment:
This condition looks a bit strange tbh. Can it be not a mnemonicGroup and what will happen in this case?

Reply:
Agree, let me try to explain how I understand this part of display logic.
Here we are already in a context where`isSignerGroup = true`, which means that it is a wallet created/imported using either mnemonic phrase or a private key.
And now instead of having
```
to: `/backup-wallet?groupId=${walletGroup.id}&backupKind=verify`,
onClick: needsBackup(walletGroup) ? openInTabView : undefined,
```
we have
```
to: isMnemonicGroup ? `/backup?groupId=${walletGroup.id}` : undefined,
onClick: isMnemonicGroup
  ? (event) =>
    openHref(event, {
      windowType: 'tab',
      windowLayout: 'page',
    }) : undefined,
```
So basically nothing have changed significantly, except that I fixed a couple of bugs here (user was unable to backup already backedup wallet and missing `needsBackup` condition for the "to" attribute)

Reply:
I used `isMnemonicGroup` instead of `needsBackup` here because I believe users should have the option to back up their wallet, even if it was imported or already backed up. If you have any concerns or think otherwise, let's discuss

Reply:
1.  We do not show group view for PrivateKeyWalletContainer. How to see these conditionals in effect?
2. Let's do a helper for
```
const openInTabView = (event) => openHref(event, { windowType: 'tab', windowLayout: 'page' })
```
The diff will be smaller and easier to read

3. 
> I believe users should always have the option to back up their wallet

I agree 100%, but we went over this with product already and currently they disagree. Let's not introduce such changes unexpectedly

Reply:
1. Updated, not relevant anymore
2. Done ✅ 
3. I was like 100% sure it is a bug, will try to highlight changes like that next time so there're easier to spot & discuss